unit_tests/test_tlslite_extensions.py

```diff
@@ -1703,7 +1703,7 @@ def test_extData_none_mode(self):
     def test_extData_mode(self):
         ext = HeartbeatExtension().create(0)
 
-        self.assertEqual(ext.extData, '\x00')
+        self.assertEqual(ext.extData, b'\x00')
```

Comment:
you'll need to [rebase and fixup](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) those commits before force pushing them for the CI to pass